api/simplyrets-openapi.yaml

```diff
@@ -1576,6 +1578,27 @@ components:
         type: array
         items:
           type: string
+    pool:
+      required: false
+      in: query
+      name: pool
+      description: >
+        Filter listings on pool features (`.property.pool`). There
+        are multiple ways to use this parameter:
+
+        - `pool=true`: Return listings where `.property.water` is NOT
```

Comment:
These should be property.pool - line 1589 and 1591 both

Reply:
@spmiller2 good catch, thank you. I fixed both occurrences.